.gitignore

```diff
@@ -93,6 +93,12 @@ unlinked_spec.ds
 **/ios/ServiceDefinitions.json
 **/ios/Runner/GeneratedPluginRegistrant.*
 
+# macOS
+**/macos/Flutter/GeneratedPluginRegistrant.swift
```

Comment:
Wonder if ios being in `Runner` and macos being in `Flutter` means this is ready for a future cleanup?. Maybe `**/macos/**/GeneratedPluginRegistrant.*` to make this a bit more refactor-proof.

cc @jmagman

Reply:
I'm skeptical of trying to future proof gitignores too much, you might end up accidentally ignoring real code in the future.